### ISSUE TITLE
Perfromance degradation issue

### ISSUE THREAD

**Issue by RohitKeshari** [TNTWEN/OpenVINO-YOLOV4#24]
I am getting less performance after conversion. Have you bench mark your code with before conversion and after conversion?

**Comment by TNTWEN** [comment]
In order to test the accuracy of yolov4 on the embedded platform after model conversion under different frameworks. AlexeyAB provided test pictures, and I also uploaded my test results in the README.In some tensorflow1.x,tensorflow2.x and pytorch projects, after the model conversion, the accuracy dropped significantly. In my OpenVINO implementation, the accuracy is not much different from darknet, and even better than darknet in some scenarios.


Have you added --reverse_input_channels  when you run mo.py?
And have you set the same confidence threshold when you test your model？ In object_detection_demo_yolov3_async.py,you could set -t 0.2  , (default -t is 0.5)

**Comment by RohitKeshari** [comment]
Thanks for your reply. I have used --reverse_input_channels while converting yolov4-tiny model. yolov3-tiny performance is matches with before and after conversion performance. I have used https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_YOLO_From_Tensorflow.html for the conversion. Regarding yolov4-tiny I am getting reduced performance around 25-30%. I was wondering If you have compared results on full data. 

Moreover, the test results in your README has different confidence score before and after conversion.
